src/shogun/multiclass/LSHKNNSolver.h

```diff
@@ -16,9 +16,9 @@ namespace shogun
 {
 
 /**
- * The LSHKNNSolver class is a solver class which inherit from KNNSolver class.
+ * The LSHKNNSolver class is a solver class.
  * It use LSH(short for Locality-sensitive hashing) to do the nearest neighbor search.
- * More detail information can be found in https://en.wikipedia.org/wiki/Locality-sensitive_hashing.
+ * More detailed information can be found in https://en.wikipedia.org/wiki/Locality-sensitive_hashing.
```

Comment:
Can you make this unified with above references

For more information, see ...